belgif-rest-problem/src/test/java/io/github/belgif/rest/problem/validation/TimeReflectionUtilTest.java

```diff
@@ -49,20 +49,27 @@ void nokWithInvalidType() {
                 "Should have thrown exception");
     }
 
```

Comment:
This test class still uses JUnit assertions for validating an exception was thrown